### ISSUE TITLE
[Migration] Verify compatible CPU & hypervisor capabilities across migration

### ISSUE THREAD

**Issue by balamuruhans** [autotest/tp-libvirt#1638]
Explore and add scenarios to verify CPU compat migration to support POWER 8 <-> POWER9 migration and to verify Hypervisor capabilities, need to do some background work for this.

**Comment by balamuruhans** [comment]
@dzhengfy you are right! I worked on compat mode migration on the PR mentioned. I have to do some exploration on this, it would be great if you can help or suggest me on this.

**Comment by dzhengfy** [comment]
In x86_64 machines, the cpu features are considered to be checked before and after migration, especially for cpu mode='host-model'. But this seems not applied to Power as Power cpu does not have <feature>.
